lua/nvim-tree/renderer/components/git.lua

```diff
@@ -124,6 +126,17 @@ local git_hl = {
   [" A"] = "none",
 }
 
+function M.setup_signs()
+  local i = M.icon_state.icons.git_icons
```

Comment:
this might not be initialized yet

Reply:
It should be.
`M.icon_state = _icons.get_config()` will always be called before `setup_signs`, as renderer init requires git.

Reply:
I have got
```lua
vim.g.nvim_tree_icons = {
  git = {
    unstaged = 'M',
    staged = 'S',
    unmerged = 'U',
    renamed = 'R',
    untracked = '?',
    deleted = 'D',
    ignored = 'I',
  },
}
```
but icons in `signcolumn` are (I guess) some default (x, ★). Should I open an issue for it?

Reply:
are you properly setting up the nvim_tree_icons before requiring nvim-tree ?

Reply:
Ah, thanks! I was setting those `g:` before `require('nvim-tree').setup()` until recently adding a pcall-require-guard to all my plugins. User error - all works fine now, thanks!